cmd/edl/edl.go

```diff
@@ -34,6 +34,7 @@ var (
 func main() {
 	masterURL := flag.String("master", "", "Address of a kube master.")
 	kubeConfig := flag.String("kubeconfig", "", "Path to a kube config. Only required if out-of-cluster.")
+	autoClean := flag.Bool("autoclean", false, "Auto clean pods after terminating job, default false")
```

Comment:
So make this default false means user may need to get the logs?

Reply:
Yes, if flag `autoclean` is  false, controller will maintain pods after success or failure. Otherwise,  all pods will be deleted automatically. It's useful to debug and get logs.